docker-compose.yml

```diff
@@ -0,0 +1,154 @@
+      - source: hive-kubeconfig
+        target: /app/secrets/aks.kubeconfig
+    environment:
+      # inherit from host
+      - ADMIN_OBJECT_ID
+      - ARO_IMAGE
+      - AZURE_ARM_CLIENT_ID
+      - AZURE_CLIENT_ID
+      - AZURE_CLIENT_SECRET
+      - AZURE_DBTOKEN_CLIENT_ID
+      - AZURE_ENVIRONMENT
+      - AZURE_FP_CLIENT_ID
+      - AZURE_FP_SERVICE_PRINCIPAL_ID
+      - AZURE_GATEWAY_CLIENT_ID
+      - AZURE_GATEWAY_CLIENT_SECRET
+      - AZURE_GATEWAY_SERVICE_PRINCIPAL_ID
+      - AZURE_PORTAL_ACCESS_GROUP_IDS
+      - AZURE_PORTAL_CLIENT_ID
+      - AZURE_RP_CLIENT_ID
+      - AZURE_RP_CLIENT_SECRET
+      - AZURE_SERVICE_PRINCIPAL_ID
+      - AZURE_SUBSCRIPTION_ID
+      - AZURE_TENANT_ID
+      - DATABASE_ACCOUNT_NAME
+      - DATABASE_NAME
+      - DOMAIN_NAME
+      - KEYVAULT_PREFIX
+      - LOCATION
+      - MOCK_MSI_CERT
+      - MOCK_MSI_CLIENT_ID
+      - MOCK_MSI_TENANT_ID
+      - OIDC_STORAGE_ACCOUNT_NAME
+      - PARENT_DOMAIN_NAME
+      - PARENT_DOMAIN_RESOURCEGROUP
+      - PROXY_HOSTNAME
+      - PULL_SECRET
+      - RESOURCEGROUP
+      - SECRET_SA_ACCOUNT_NAME
+      - STORAGE_ACCOUNT_DOMAIN
+
+      # override
+      - ARO_ADOPT_BY_HIVE=true
+      - ARO_CHECKOUT_PATH=/app
+      - ARO_INSTALL_VIA_HIVE=true
+      - HIVE_KUBE_CONFIG_PATH=/app/secrets/aks.kubeconfig
+      - KUBECONFIG=/app/secrets/aks.kubeconfig
+      - RP_MODE=development
+    expose:
+      - "8443"
+    ports:
+      - "127.0.0.1:8443:8443"
+    healthcheck:
+      test: ["CMD", "curl", "-k", "http://localhost:8443/healthz"]
+      interval: 30s
+      timeout: 30s
+      retries: 3
+    restart: on-failure:3
+
+  portal:
+    image: ${LOCAL_ARO_RP_IMAGE}:${VERSION}
+    container_name: aro-portal
+    depends_on:
+      rp:
+        condition: service_healthy
+    environment:
+      - RP_MODE
+      - AZURE_SUBSCRIPTION_ID
+      - AZURE_TENANT_ID
+      - LOCATION
+      - RESOURCEGROUP
+      - AZURE_PORTAL_CLIENT_ID
+      - AZURE_PORTAL_ELEVATED_GROUP_IDS
+      - AZURE_PORTAL_ACCESS_GROUP_IDS
+      - AZURE_RP_CLIENT_SECRET
+      - AZURE_RP_CLIENT_ID
+      - KEYVAULT_PREFIX
+      - DATABASE_ACCOUNT_NAME
+      - DATABASE_NAME
+      - NO_NPM=1
+    ports:
+      - "127.0.0.1:8444:8444"
+      - "127.0.0.1:2222:2222"
+    secrets:
+      - source: proxy-client-key
+        target: /app/secrets/proxy-client.key
+      - source: proxy-client-crt
+        target: /app/secrets/proxy-client.crt
+      - source: proxy-crt
+        target: /app/secrets/proxy.crt
+    cap_drop:
+      - NET_RAW
+    command: ["portal"]
+    restart: on-failure:3
+    healthcheck:
+      test: ["CMD", "curl", "-k", "http://localhost:8444/healthz"]
+      interval: 30s
+      timeout: 10s
+      retries: 3
+
+secrets:
+  proxy-client-key:
+    file: ./secrets/proxy-client.key
+  proxy-client-crt:
+    file: ./secrets/proxy-client.crt
+  proxy-crt:
+    file: ./secrets/proxy.crt
+  hive-kubeconfig:
+    file: ./aks.kubeconfig
```

Comment:
If we remove the secrets section, Docker Compose will indeed throw an error like:

`service "rp" refers to undefined secret proxy-client-key: invalid compose project`

The top-level secrets section is what tells Docker Compose where to find the actual files for the secrets. Without it, Docker doesn’t know where to look.

Reply:
my suggestion is the opposite direction - let's use compose, rather than https://github.com/Azure/ARO-RP/blob/d64782c410f1155a79047849b5d1fb8900242c28/Makefile#L444-L456 - is there a reason we couldn't delete this part of the makefile now?

Reply:
this also makes me realize we've removed the `aks.kubeconfig` dependency (which was in the podman-secrets target, and now needs to be on the `run-rp` target)

Reply:
Thanks for pointing this, I agree to remove it from Makefile 
- The `podman-secrets` target has been removed as Docker Compose now fully manages the secrets.

Reply:
I've added aks.kubeconfig as a dependency for the run-rp target